portfolio/src/main/webapp/script.js

```diff
@@ -85,16 +90,16 @@ function makeComment(heading, content) {
 
 function getDropdownVal() {
     const dropdown = document.getElementById("num-comments");
-    dropdown.onchange = changeDropdownVal;
-    if (sessionStorage["num-comments"]) {
-        dropdown.value = sessionStorage["num-comments"];
-    }   
+    const defaultComments = 10;
```

Comment:
This should probably be moved to somewhere at the top of the file or maybe in a separate config file, but it is ok to leave this here for now to not complicate this pull request even more.

Reply:
ok, i added a comment about it